app/controllers/documents_controller.rb

```diff
@@ -137,13 +143,58 @@ def update
 
   def new
     @document = Document.new
-      respond_to do |format|
-        format.js { render "documents/new" }
-      end
+    respond_to do |format|
+      format.js
+    end
   end
 
   def create
-    Document.create(:title=>"Doc1",:creator=>"dchl",:publisher=>"dchl",:type=>"text",:format=>"application/octet-stream",:source=>"enroll_system",:language=>"en",:rights=>"public")
+    @employer_profile = Organization.all_employer_profiles.where(legal_name: params[:document][:creator]).last.employer_profile
+    @employer_profile.employer_attestation= EmployerAttestation.new() unless @employer_profile.employer_attestation
+    #@employer_profile.employer_attestation.employer_attestation_doccument = EmployerAttestationDocument.new() unless @employer_profile.employer_attestation.employer_attestation_doccument
```

Comment:
you can do
`@employer_profile.employer_attestation ||= EmployerAttestation.new()`